ocs_ci/ocs/ui_selenium/const_xpath.py

```diff
@@ -0,0 +1,19 @@
+xpath_pvc_page = {
+    "Storage Tab": "/html/body/div[2]/div/div/div/div/div[1]/div/div/nav/ul/li[5]/button",
+    "PVC Page": "//*[@id='page-sidebar']/div/nav/ul/li[5]/section/ul/li[2]/a",
+    "PVC Project Selector": "//*[@id='content']/div[2]/div/div/div/button",
+    "PVC Select Project openshift-storage": "//*[@id='openshift-storage-link']",
+    "PVC Create Button": "//*[@id='yaml-create']",
+    "PVC Storage Class Selector": "//*[@id='storageclass-dropdown']",
+    "ocs-storagecluster-ceph-rbd": "//*[@id='ocs-storagecluster-ceph-rbd-link']/span",
+    "ocs-storagecluster-ceph-fs": "//*[@id='ocs-storagecluster-cephfs-link']/span",
+    "PVC Name": "//*[@id='pvc-name']",
+    "ReadWriteOnce": "//*[@id='content-scrollable']/div/form/div[1]/div[3]/label[1]/input",
+    "ReadWriteMany": "//*[@id='content-scrollable']/div/form/div[1]/div[3]/label[2]/input",
+    "ReadOnlyMany": "//*[@id='content-scrollable']/div/form/div[1]/div[3]/label[3]/input",
+    "PVC Size": "//*[@id='request-size-input']",
+    "PVC Label Selector": "//*[@id='content-scrollable']/div/form/div[1]/div[5]/div/label/input",
+    "PVC Create": "//*[@id='save-changes']",
+    "PVC Actions": "//*[@id='content-scrollable']/div[1]/h1/div[2]/div/button",
+    "PVC Delete": "//*[@id='content-scrollable']/div[1]/h1/div[2]/div/ul/li[7]/button",
```

Comment:
Are these "Recorded" from UI by Selenium?

Reply:
this is XPath.
XPath is a technique in Selenium to navigate through the HTML structure of a page.